test/BuiltIns/rlexe.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<regress-exe>
+  <test>
+    <default>
+      <files>ChakraLib.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
+      <tags>BuiltIn LdChakraLib</tags>
```

Comment:
That's right. I've made a new unit test to make it clearer and fixed the arguments.

---
In reply to: [115563779](https://github.com/Microsoft/ChakraCore/pull/2898#discussion_r115563779) [](ancestors = 115563779)